Prac05/count_words.py

```diff
@@ -0,0 +1,32 @@
+"""
+Program asks user for a string containing several words, then counts how many times each word appears in the string.
+Text: this is a collection of words of nice words this is a fun thing it is
+"""
+
+user_string = input("Enter the words for your string: ")
+string_list = user_string.split()
```

Comment:
split() by default does split(" ") which is ok but I think it's better to show what you are splitting on